### PR TITLE
FIX / Added info card component for Profile information

### PR DESCRIPTION
**Pull request by cody-daigle**

### CODE REVIEW

File: client/src/components/profile/Blogs.tsx [hunk 1]
```diff
@@ -6,16 +6,13 @@ import Typography from '@mui/material/Typography';
 import Grid from '@mui/material/Grid';
 import Card from '@mui/material/Card';
 import CardMedia from '@mui/material/CardMedia';
-import Link from '@mui/material/Link';
-import { User } from '@prisma/client';
-import Box from '@mui/material/Box';
+import { Link } from 'react-router-dom';
 
 interface Blog {
```
Comment (AlexPHebert2000) [inline client/src/components/profile/Blogs.tsx:12]: Could this be imported from prisma client?

File: client/src/components/profile/Blogs.tsx [hunk 2]
```diff
@@ -26,45 +23,46 @@ const Blogs = ({ devId, mediumId }: UserProps): ReactElement => {
...
         })
         .catch((err) => console.error(err));
     };
-
-    if (devId) {
-      getDev();
-    }
     if (mediumId) {
       getMedium();
+    } else if (devId) {
```
Comment (AlexPHebert2000) [inline client/src/components/profile/Blogs.tsx]: Does this limit the user to choosing one or the other?

**Review by AlexPHebert2000 — APPROVED** [review]

File: client/src/components/profile/Blogs.tsx (hunk 2 above)
Comment (cody-daigle) [reply client/src/components/profile/Blogs.tsx]: Yes, I didn't notice the old changes took over. Thank you.
I corrected it and pushed the updated version of it.

File: client/src/components/profile/Blogs.tsx (hunk 1 above)
Comment (cody-daigle) [reply client/src/components/profile/Blogs.tsx:12]: Changed it to utilize MUI instead. I must have accepted the wrong changes.

**Review by Hackman78 — APPROVED** [review]

**Pull request merged**